packages/x-components/src/x-modules/tagging/store/types.ts

```diff
@@ -54,7 +55,13 @@ export interface TaggingMutations {
  *
  * @public
  */
-export interface TaggingActions {}
+export interface TaggingActions {
+  /**
+   * Tracks the tagging of the query.
```

Comment:
We're tracking the query of XC components as this action is launched when the event `SearchTaggingChanged` is emitted, and this event is emitted if the query tagging changes.

It refers to the type of tagging we're tracking instead of what information we're tracking.

Reply:
I think what @javieri-empathy is referring to is that this is a generic action that's going to be used to track anything, not only the query. As you did it, you know it just receives a `taggingInfo` object which does not care about the user interaction event that triggered it.

Reply:
```suggestion
   * Tracks a user interaction.
```